pkg/datasources/network_policies_acceptance_test.go

```diff
@@ -0,0 +1,200 @@
+package datasources_test
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"regexp"
+	"testing"
+
+	acc "github.com/Snowflake-Labs/terraform-provider-snowflake/pkg/acceptance"
+	"github.com/Snowflake-Labs/terraform-provider-snowflake/pkg/acceptance/helpers/random"
+	"github.com/Snowflake-Labs/terraform-provider-snowflake/pkg/acceptance/testenvs"
+	"github.com/Snowflake-Labs/terraform-provider-snowflake/pkg/provider/resources"
+	"github.com/Snowflake-Labs/terraform-provider-snowflake/pkg/sdk"
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+	"github.com/hashicorp/terraform-plugin-testing/tfversion"
+	"github.com/stretchr/testify/require"
+)
+
+func TestAcc_NetworkPolicies_Complete(t *testing.T) {
+	_ = testenvs.GetOrSkipTest(t, testenvs.ConfigureClientOnce)
+
+	id := acc.TestClient().Ids.RandomAccountObjectIdentifier()
+	id2 := acc.TestClient().Ids.RandomAccountObjectIdentifier()
+	comment := random.Comment()
+
+	databaseId := acc.TestClient().Ids.RandomAccountObjectIdentifier()
+	schemaId := acc.TestClient().Ids.RandomDatabaseObjectIdentifierInDatabase(databaseId)
+
+	allowedNetworkRuleId1 := acc.TestClient().Ids.RandomSchemaObjectIdentifierInSchema(schemaId)
```

Comment:
do we need to pre-create ids?

Reply:
Yeah, I wanted to use them in the asserts and preconfig creating network rules